slayer_rails.gemspec

```diff
@@ -21,10 +21,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency('slayer', '~> 0.2')
+  spec.add_dependency 'slayer', '~> 0.2'
```

Comment:
I know this will _usually_ be run in a rails environment, but we should technically list activemodel in our dependencies, right?